### DISCUSSION TITLE
请问这个本地验证码如何配置的？我这里没有显示

### DISCUSSION THREAD

**Discussion by thegame-hhh** [mikumifa/biliTickerBuy#288] [提问专区]
![捕获](https://github.com/mikumifa/biliTickerBuy/assets/174237832/df3a689c-1a55-4363-be56-e5dffc76b135)

**Comment by mikumifa** [comment]
你的机器不支持所有没有显示

**Reply by thegame-hhh** [reply]
谢谢
